build/demo-launcher/src/launcher/main.py

```diff
@@ -272,9 +275,18 @@ async def boot(scheduler: Scheduler):
         client = await sshClient(
             username, sshkey_private, credentials.passphrase, sshkey_cert_public
         )
-        sinfo = SinfoVersionCommand()
-        version = await client.execute(sinfo)
-        scheduler_campatible = Version(version) > Version("22.05")
+
+        if scheduler.scheduler_type == "pbs":
+            qstat = QstatVersionCommand()
+            version = await client.execute(qstat)
+            scheduler_campatible = Version(version) == Version("23.06.06")
```

Comment:
Does it need to be exactly 23.06.06?

Reply:
Additionally, in the future, we should change the way we test the scheduler presence with an abstract class that includes the method